src/main/java/pro/trevor/tankgame/util/IRandom.java

```diff
@@ -0,0 +1,36 @@
+package pro.trevor.tankgame.util;
+
+public interface IRandom extends IJsonObject {
+
+    // These are technically implementation details, but could be useful to multiple implementers of IRandom.
+    double DOUBLE_UNIT = 0x1.0p-53d; // 1.0d / (1L << Double.PRECISION)
+    float FLOAT_UNIT = 0x1.0p-24f; // 1.0f / (1 << Float.PRECISION)
+
+    void setSeed(long seed);
+
+    // Either true or false
```

Comment:
Can you make the comments on these functions java doc I think it could be useful to generate them at some point.